src/index.html

```diff
@@ -17,6 +29,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo_image"
+          alt="Moyo Company Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_link">
+          <li class="nav_item">
+            <a
+              class="nav_list is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#Smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
```

Comment:
The class name 'nav_list' does not accurately represent the content of the element. It's better to use a class name like 'nav__link'.